Update Report documentation (#51)
gatling/frontline-cloud-doc: content/reference/user/reports/index.md

This page consists of:

- [The run bar]({{< ref "#run-bar" >}})
- [The top navigator menu]({{< ref "#top-navigator-menu" >}})
- [Assertions]({{< ref "#assertions" >}})
- [The timeline]({{< ref "#timeline" >}})
- [Tabs]({{< ref "#tabs" >}})
- [Filter bar]({{< ref "#filter-bar" >}})
- [Charts area]({{< ref "#charts" >}})
- [The summary]({{< ref "#summary" >}}) (only for requests and groups tabs)
- [Public links]({{< ref "#public-links" >}})
- [Export PDF]({{< ref "#export-pdf" >}})
- [Tips]({{< ref "#tips" >}})

{{< img src="reports.png" alt="Reports" >}}

### Run Bar

This bar is a combination of buttons:

- **Start / Stop**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
- **Generate public link**: To create a [public link]({{< ref "../reports#public-links" >}})
- **Export**: To [export]({{< ref "../reports#export" >}}) reports

{{< img src="run-bar.png" alt="Run bar" >}}

### Top Navigator Menu {#top-navigator-menu}

The navigation menu enable you to choose the simulation time range.

{{< img src="timewindow.png" alt="Timewindow" >}}

### Assertions {#assertions}

> The label below is used to display the status of the simulation (Ongoing, successful, timeout...).

Reviewer: ```suggestion
The label below shows the status of the simulation (Ongoing, successful, timeout...).
```

> If your simulation has assertions, this label will be clickable to show the assertions results.

Reviewer: ```suggestion
If your simulation has assertions, you can click this label to show the assertions results.
```

> 

Reviewer 2: ```suggestion
### Top Navigator Menu
The navigator menu enables you to choose the simulation time range.
```
Line 31.

This bar is a combination of buttons:

> - **Start / Abort**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)

Reviewer 3: You should describe the run bar above the tabs, as we describe this page from top to bottom

Author: done

- **Generate public link**: To create a public link
- **Export**: To [export]({{< ref "../reports#export" >}}) reports

{{< img src="run-bar.png" alt="Run bar" >}}

> ### Sub Run Bar

Reviewer 3: This is not a run bar, as we don't run simulation from here, maybe tab bar?

Reviewer 2: The purpose of those buttons is to filter the data present in the chart, maybe the `filter bar`?

Reviewer 3: yes, that's a better name

This bar is a combination of buttons:

- **Switch to Summary**: Switch to [summary]({{< ref "../reports#summary" >}}) view for Requests & Groups tabs
- buttons to filter the metrics drawn in the charts area

{{< img src="sub-run-bar.png" alt="Run bar" >}}

### Generate Public Links

> 

Reviewer 2: Replace `Frontline` with `Gatling Enterprise`.

{{< img src="timeline.png" alt="Timeline" >}}

### Tabs

Below the navigator chart, there are tabs to switch charts.
Each tab has the same structure except the summary that is available only for requests and groups tabs.

{{< img src="tabs.png" alt="Tabs" >}}

### Filter Bar {#filter-bar}

This filter bar is a combination of buttons:

- **Switch to Summary**: Switch to [summary]({{< ref "../reports#summary" >}}) view for Requests & Groups tabs
- buttons to filter the metrics drawn in the charts area

{{< img src="filter-bar.png" alt="Filter bar" >}}

### Charts

> Each charts in Gatling Enterprise are connected to each other, so if you select a time window on a chart it will automatically change it

Reviewer: ```suggestion
Each chart in Gatling Enterprise is connected to each other, so if you select a time window on a chart it will automatically change it
```

Reviewer: ```suggestion
All charts in Gatling Enterprise are connected to each other, so if you select a time window on a chart it will automatically change it
```

{{< img src="summary.png" alt="Summary" >}}

### Generate Public Links {#public-links}

> A public link is a link of the current reports which will be accessible to anyone, without having to log-in to FrontLine. To generate a public link, click on the *Generate public link* button and choose the expiration date of your link.

Reviewer 4: ```suggestion
A public link is a link of the current report which will be accessible to anyone, without having to log-in to Gatling Enterprise. To generate a public link, click on the *Generate public link* button and choose the expiration date of your link.
```

{{< img src="generate-public-link.png" alt="Generate public links" >}}

The maximum allowed lifetime for a public link is 1 year.

> Once you have chosen an expiration date, clicking on the generate button.

Reviewer: ```suggestion
Chose an expiration date, then click the generate button.
```

Reviewer: ```suggestion
Choose an expiration date, then click on the generate button.
```

> Once you have chosen an expiration date, you can proceed by clicking on the generate button.

Reviewer 4: ```suggestion
Once you have chosen an expiration date, click on the generate button.
```

{{< img src="successful-generation-public-link.png" alt="Successful public link generation" >}}

> You can copy the public link to share your reports to non-FrontLine users, or click on the "Go" Button to access it yourself. You can click on the "OK" button to close this modal.

Reviewer 4: ```suggestion
You can copy the public link to share your reports to non-Gatling Enterprise users, or click on the "Go" Button to access it yourself. You can click on the "OK" button to close this modal.
```

### Export PDF {#export-pdf}

> When clicking on the blue export button in the navigator menu, you will have access to a page where you can configure and then export a PDF report of a specific simulation.

Reviewer 4: ```suggestion
When clicking on the blue export button in the navigator menu, you will have access to a page where you can configure and then export a PDF report of a specific simulation run.
```

{{< img src="summary.png" alt="Summary" >}}

### Export PDF {#export}

> When clicking on the green button in the navigation bar, you will have access to a page where you can configure and then export a PDF report of a specific simulation.

Reviewer 2: ```suggestion
When clicking on the green button in the navigation bar, you will have access to a page where you can configure and then export a PDF report of a specific simulation.
```

`Navigation bar` is already use in the documentation for the NavBar (left bar).

- **Text Area**: add an editable text element.
- **New Page**: allow you to skip a page in the report.
- **Run**:
  - **Status**: add an editable text element with a predefined text set to the status of the selected run.
  - **Comments**: add an editable text element with a predefined text set to the comments of the selected run.
  - **Assertions**: add a table with the assertions of the selected run.
  - **Summary**: add the summary table of the selected run in a new landscape page.
- **Chart**: add a chart element that you can interact with before exporting it to PDF.

> - **Counts**: add a count chart element that you can interact with before exporting it to PDF.

Reviewer: third singular

In the top right menu, you can activate the **Date Time** setting to be able to switch from elapsed time to date time.

{{< img src="multiplecheck-date.png" alt="Multiple check" >}}
{{< img src="multiple-date.png" alt="Multiple" >}}

#### Highlight Legend

> By hovering the label on the percentiles chart legend, you will be able to highlight the curve on the chart, leading to a better view of that curve.

Reviewer 4: ```suggestion
If you hover your mouse over a label on the percentiles chart legend, you will highlight the curve on the chart, leading to a better view of that curve.
```

Reviewer 4: Also, line 219 : "The highlight legend option is enabled for every "non stacked" graph."
